website/content/en/docs/deployment/connect-kubeflow-dashboard.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "Connect to your Kubeflow Dashboard"
+description = "Start experimenting and running your end-to-end ML workflows with Kubeflow on AWS"
+weight = 70
++++
+
+## Port-forward (Manifest deployment)
+
+### Option 1: Amazon EC2 
+
+Run the following command on your EC2 instance: 
+```sh
+make port-forward
+```
+
+Then,on your local machine, run the following:
+
+```sh
+ssh -i <path-to-your-key-pem> -L 8080:localhost:8080 -N ubuntu@ec2-<Public-IPv4-DNS>.compute-1.amazonaws.com -o ExitOnForwardFailure=yes
+```
+
+You can then open the browser and go to [http://localhost:8080/](http://localhost:8080/).
+
+### Option 2: Docker 
+
+Run the following command:
+```sh
+make port-forward IP_ADDRESS=0.0.0.0
+```
+
+You can then open the browser and go to [http://localhost:8080/](http://localhost:8080/).
+
+### Option 3: AWS Cloud9
+
+Connect to your Kubeflow dashboard with a single command:
+```sh
+make port-forward
+```
+
+Then, open the AWS Cloud9 browser and go to [http://localhost:8080/](http://localhost:8080/).
+
+## Port-forward (Terraform deployment)
+
+First, update the `kubeconfig`:
+```sh
+$(terraform output -raw configure_kubectl)
```

Comment:
should this be `terraform output -raw configure_kubectl` instead of `$(terraform output -raw configure_kubectl)`

@rrrkharse